dyninstAPI/src/baseTramp.C

```diff
@@ -299,6 +299,11 @@ bool baseTramp::generateCodeInlined(codeGen &gen,
    std::vector<AstNodePtr> miniTramps;
 
    if (point_) {
+      // Stable sort snippets so that Prologues < Regular snippets < Epilogues.
```

Comment:
This comment is too literal- it just tells me what `stable_sort` does. I would say something along the lines of "Order snippets by their type with prologues before regular snippets and epilogues at the end without changing the order of snippets of the same type."